uPortal-portlets/src/main/java/org/apereo/portal/portlets/search/SearchPortletController.java

```diff
@@ -619,6 +621,12 @@ public ModelAndView showSearchResults(
         if (portalSearchResults != null) {
             results = portalSearchResults.getResults();
         }
+
+        results.forEach(
+                (key, value) -> {
+                    value.sort(Comparator.comparingInt(tuple -> tuple.first.getRank()));
```

Comment:
Is the intent to sort the contents of value?
or the results as a whole?

Reply:
Apparently the contents of `value`.

This is originally a solution from @narcolapser;  I translated it to uP5.

Reply:
Here's an example of what the `ConcurrentMap` contains:

```
{
  portal.results=[
    Tuple [
      first=org.apereo.portal.search.SearchResult@5a8a2248, 
      second=/uPortal/normal/render.uP?pCt=logging-in.ctf13
    ], 
    Tuple [
      first=org.apereo.portal.search.SearchResult@371416a9, 
      second=/uPortal/p/directory.ctf3/max/render.uP?pCm=view&pP_action=findByUsername&pP_username=student
    ], 
    Tuple [
      first=org.apereo.portal.search.SearchResult@68931490, 
      second=/uPortal/f/u30l1s100/p/student-feature.u30l1n122/max/render.uP
    ],
    Tuple [
      first=org.apereo.portal.search.SearchResult@654598b1, 
      second=/uPortal/f/news-fav-collection/p/announcements.u22l1n7/max/render.uP
    ]
  ]
}
```

The keys represent tabs on the search results screen.  (Results are divided into categories.)

The value associated with each key is the list of results that match the category.